### PR TITLE
Fix updating cloudformation script

### PR DESCRIPTION
**Pull request by evilmarty**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -54,10 +54,10 @@ steps:
       ROOT_VOLUME_SIZE: 500
       SCALE_UP_ADJUSTMENT: 1
       AGENTS_PER_INSTANCE: 5
-      SPOT_PRICE: 0.2670
+      SPOT_PRICE: 0.2660
```
Comment (langsharpe) [inline .buildkite/pipeline.yml:57]: I like that we let the spot instance market determine if we run our builds. 🤣

**Review by langsharpe — APPROVED** [review]

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (evilmarty) [reply .buildkite/pipeline.yml:57]: Are you not a gambling man?

Comment (elliot) [reply .buildkite/pipeline.yml:57]: We've been running spot instances on all but the build node for over a month 😄 

The bid prices are AZ specific though, so even if it hit the on demand, the ASG would bring up nodes in a different zone.

**Comment by elliot** [comment]
Nice work 👍

**Pull request merged**